src/modules/Dropdown/Dropdown.js

```diff
@@ -354,9 +356,9 @@ export default class Dropdown extends Component {
 
   handleSearchKeyDown = (e) => {
     // console.debug('Dropdown.handleSearchKeyDown()')
-    // console.log(e.key)
-    switch (e.key) {
-      case 'ArrowDown':
+    // console.log(keyboardKey.getCode(e))
+    switch (keyboardKey.getCode(e)) {
+      case keyboardKey.Spacebar:
```

Comment:
This method was actually cruft, removed.  The dropdown is opened anytime there is a query.  The document listeners handle the arrow down key as well.